src/app/components/LiveRegion/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import { useLiveRegionContext } from './LiveRegionContext';
+
+const LiveRegion = () => {
+  const { liveRegionItem } = useLiveRegionContext();
```

Comment:
Do we think we need guarding around this? To prevent people from using this component without having a provider encapsulating it?

Reply:
Is there an error that gets thrown if the form isn't wrapped in the LiveRegionContext?

Reply:
Yeah an error gets thrown saying `liveRegionItem` doesn't exist.

Reply:
I think if the error results in the application not rendering then thats probably a good thing? Anyone making use of that component outside of it being in a context provider will see the error, so the code wouldn't reach production. 

I could be going about this the wrong way though. Was there a guard you had in mind?

Reply:
I was thinking some lint thing to show up in vs code, now thinking about it, don't think it's possible.

Maybe adjusting the error to be more meaningful, adding this to the component:

```jsx
if (!liveRegionItem) {
   throw new Error('You cannot render the LiveRegion without having a LiveRegionProvider higher up in the component hierarchy')
}
```